docs/audit_report/src/02_security_issues.rst

```diff
@@ -3,3 +3,5 @@ Security and Vulnerabilities
 
 Currently, we are not aware of any security issues that needed to be addressed
 between |botan_git_base_ref| and |botan_version|.
+
+.. todo:: Update this in case some vulnerability is found and fixed before the 3.3.0 release.
```

Comment:
Maybe we want a very brief list/description for us what we do for each release?

I.e.:
- Prepare documents for new release (i.e. the changes you are doing in this PR).
- Then finalizing the release, check that all the TODOs added in the prepare documents are done.
- ...

Probably not something for this PR, but would be helpful so that we don't forget one of the remaining manual steps.

Reply:
Yeah, some sort of release checklist would certainly be helpful. I'll cook something up in another PR.